### PR TITLE
Adding a warning section in our documentation

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1622820

### CODE REVIEW

**Comment by mburke5678** [comment]
@fasho PTAL

File: install_config/install/advanced_install.adoc [hunk 1]
```diff
@@ -2489,10 +2489,22 @@ specific components as needed. By breaking up the roles and playbooks, there is
...
 
-[NOTE]
-====
 The playbooks and their ordering are detailed below in
 xref:running-the-advanced-installation-individual-components[Running Individual Component Playbooks].
+
+
+[NOTE]
+====
+After running the installation, due to a known issue, the following directories might get created:
```
Comment (sdodson) [inline install_config/install/advanced_install.adoc]: I'd clarify this.
`If NFS volumes are provisioned for any component, all of the following directories will be created whether their components are being deployed to NFS volumes or not.`

Comment (mburke5678) [reply install_config/install/advanced_install.adoc]: @sdodson Thank you!

**Comment by mburke5678** [comment]
@anpingli PTAL

**Comment by mburke5678** [comment]
@anpingli PTAL

cc. @vikram-redhat

**Comment by mburke5678** [comment]
@anpingli 
I am not sure what you  mean by your comment above. Are you suggesting this note is not needed?

cc. @sdodson

**Comment by mburke5678** [comment]
Changes for enterprise-3.10 and 3.11 in follow-up as Advanced Install has been removed.

**Comment by anpingli** [comment]
LGTM now

**Comment by mburke5678** [comment]
@openshift/team-documentation Small change PTAL

File: install_config/install/advanced_install.adoc [hunk 2]
```diff
@@ -2489,10 +2489,23 @@ specific components as needed. By breaking up the roles and playbooks, there is
...
+the following directories might be created whether their components are being deployed to NFS volumes or not:
+
+* *_/exports/logging-es_*
+* *_/exports/logging-es-ops/_*
+* *_/exports/metrics/_*
+* *_/exports/prometheus_*
+* *_/exports/prometheus-alertbuffer/_*
+* *_/exports/prometheus-alertmanager/_*
+
+You can delete these files after installation, if needed.
```
Comment (kalexand-rh) [inline install_config/install/advanced_install.adoc]: s/files/directories
I'd remove "if needed." /shrug

**Review by kalexand-rh — COMMENTED** [review]
One small change and a suggestion. LGTM

File: install_config/install/advanced_install.adoc (hunk 2 above)
Comment (mburke5678) [reply install_config/install/advanced_install.adoc]: Thanks!!

**Pull request merged**